OneMoreTray/ScanningJob.cs

```diff
@@ -28,8 +28,6 @@ public ScanningJob()
 		{
 			logger = Logger.Current;
 
```

Comment:
Why remove this logic from ScanningJob? Culture is thread-specific and not always inherited implicitly from parent threads, so it's set explicitly here.

It's intended to get the culture from the OneMore language settings, just like the rest of OneMore. By removing this code, now the OneMoreTray assumes the default language of Windows, not of OneMore. That breaks the case where a user wants a different language in OneNote than they do in Windows.